Core/Core/Models/Base.cs

```diff
@@ -286,4 +274,35 @@ public Base ShallowCopy()
 
     return myDuplicate;
   }
+
+  #region Obsolete
+  /// <inheritdoc cref="GetId(bool)"/>
+  [Obsolete("Serializer v1 is now deprecated, use other overload(s)")]
+  public string GetId(SerializerVersion serializerVersion)
```

Comment:
Oops. I accidently included some serialzierv1 deprecation.
I can unstage this if preferred.

Reply:
Hmmm... if it's not a huge ask lets roll this back and open an extra PR for this 🙏🏼

Reply:
reverted